spi_nand_flash/src/dhara_glue.c

```diff
@@ -178,9 +178,20 @@ int dhara_nand_is_free(const struct dhara_nand *n, dhara_page_t p)
     return 0;
 }
 
-static int is_ecc_error(uint8_t status)
+#define PACK_2BITS_STATUS(status, bit1, bit0)         ((((status) & (bit1)) << 1) | ((status) & (bit0)))
+#define PACK_3BITS_STATUS(status, bit2, bit1, bit0)   ((((status) & (bit2)) << 2) | (((status) & (bit1)) << 1) | ((status) & (bit0)))
+
+static ecc_status_t is_ecc_error(uint8_t status, uint8_t ecc_status_reg_len_in_bits)
```

Comment:
"is_..." prefix is usually used for functions which return true/false.

Here it sounds more like "get_ecc_status" or "extract_ecc_status".